tests/integration/mock_test.ts

```diff
@@ -48,7 +48,8 @@ Rhum.testPlan("mock_test.ts", () => {
         .mock(TestObject)
         .create();
       Rhum.asserts.assertEquals(
-        mock.protected_property,
+        // deno-lint-ignore no-explicit-any, eslint-ignore-next-line no-explicit-any
+        (mock as any).protected_property,
```

Comment:
Not as pretty as with previous any, but works like in jasmine: https://github.com/DefinitelyTyped/DefinitelyTyped/issues/14811

Reply:
I think in that case, the data could be anything and everything, it could be a number, string, array etc